### PR TITLE
Add "Sylvania LIGHTIFY Contact and Temperature Smart Sensor" fingerprint

### PR DESCRIPTION
**Pull request by brianegge**

### CODE REVIEW

File: devicetypes/smartthings/smartsense-open-closed-sensor.src/smartsense-open-closed-sensor.groovy [hunk 1]
```diff
@@ -16,7 +16,7 @@
 import physicalgraph.zigbee.clusters.iaszone.ZoneStatus
 
 metadata {
-	definition(name: "SmartSense Open/Closed Sensor", namespace: "smartthings", author: "SmartThings", runLocally: true, minHubCoreVersion: '000.017.0012', executeCommandsLocally: false) {
+	definition(name: "SmartSense Open/Closed Sensor", namespace: "smartthings", author: "SmartThings", runLocally: false, minHubCoreVersion: '000.017.0012', executeCommandsLocally: false) {
```
Comment (tpmanley) [inline devicetypes/smartthings/smartsense-open-closed-sensor.src/smartsense-open-closed-sensor.groovy:19]: @brianegge , thanks for the PR! We actually want to keep `runLocally` set to `true` so the DTH continues to run locally where possible so you can revert that change and just leave the new fingerprint.

Comment (brianegge) [reply devicetypes/smartthings/smartsense-open-closed-sensor.src/smartsense-open-closed-sensor.groovy:19]: I don't understand why I had to change the run local to false in order to load the IDE. Anyway, I had placed the change in it's own commit. Are you able to cherry pick just the second commit or manually copy the fingerprint into the repo?

Comment (tpmanley) [inline devicetypes/smartthings/smartsense-open-closed-sensor.src/smartsense-open-closed-sensor.groovy:19]:
```suggestion
	definition(name: "SmartSense Open/Closed Sensor", namespace: "smartthings", author: "SmartThings", runLocally: true, minHubCoreVersion: '000.017.0012', executeCommandsLocally: false) {
```

Comment (tpmanley) [reply devicetypes/smartthings/smartsense-open-closed-sensor.src/smartsense-open-closed-sensor.groovy:19]: Tried to do it with Github's "suggested change" feature but apparently I don't have permission 😁 I can just cherry-pick and open a new PR

**Comment by tpmanley** [comment]
I opened a new PR with just the new fingerprint https://github.com/SmartThingsCommunity/SmartThingsPublic/pull/3833

**Comment by brianegge** [comment]
Thank you!

**Pull request closed without merging**